shared/middleware/helpers/firebase_helper.rb

```diff
@@ -26,19 +26,48 @@ def table_as_csv(table_name)
     table_to_csv(records, column_order: ['id'])
   end
 
+  def number?(num)
+    !Float(num).nil? rescue false
+  end
+
+  def csv_as_table(csv_data)
+    records = {}
+    id = 1
+    table = CSV.parse(csv_data, headers: true)
+    table.each do |row|
+      record = OpenStruct.new
+      record.id = id
+      table.headers.each do |col|
+        value = (number? row[col]) ? row[col].to_f : row[col]
+        record[col] = value
+      end
+      records[id] = record.to_h.to_json
+      id += 1
```

Comment:
super nit: in ruby you can `id++`

Reply:
I don't think you can? I'm getting a parse error, and found this: https://stackoverflow.com/questions/3660563/why-doesnt-ruby-support-i-or-i-increment-decrement-operators

Reply:
lol woops, ignore me